consensus/hotstuff/timeoutaggregator/timeout_collectors_test.go

```diff
@@ -0,0 +1,158 @@
+package timeoutaggregator
+
+import (
+	"errors"
+	"fmt"
+	"sync"
+	"testing"
+
+	"github.com/gammazero/workerpool"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+	"go.uber.org/atomic"
+
+	"github.com/onflow/flow-go/consensus/hotstuff"
+	"github.com/onflow/flow-go/consensus/hotstuff/mocks"
+	"github.com/onflow/flow-go/module/mempool"
+	"github.com/onflow/flow-go/utils/unittest"
+)
+
+var factoryError = errors.New("factory error")
+
+func TestTimeoutCollectors(t *testing.T) {
+	suite.Run(t, new(TimeoutCollectorsTestSuite))
+}
+
+// TimeoutCollectorsTestSuite is a test suite for isolated testing of TimeoutCollectors.
+// Contains helper methods and mocked state which is used to verify correct behavior of TimeoutCollectors.
+type TimeoutCollectorsTestSuite struct {
+	suite.Suite
+
+	mockedCollectors map[uint64]*mocks.TimeoutCollector
+	factoryMethod    NewCollectorFactoryMethod
+	collectors       *TimeoutCollectors
+	lowestLevel      uint64
```

Comment:
```suggestion
	lowestView      uint64
```
This is equivalent to lowest retained view right? It's clearer for me to use the view terminology